tests/errors/notFoundApiError.ts

```diff
@@ -0,0 +1,8 @@
+export default class NotFoundApiError extends Error {
+  status: number
+
+  constructor() {
+    super()
+    this.status = 404
+  }
+}
```

Comment:
Do we need a specialist error to throw a 404?

Reply:
From memory (it's been a bit) - when testing something throwing an API i couldn't create the error that superagent throws, so i had to make my own so you could do `error.status` as it should be